test/metric-service-client-credentials.ts

```diff
@@ -91,7 +91,7 @@ describe('Bigtable/MetricServiceClientCredentials', () => {
     class FakeMetricServiceClient {
       constructor(options: ClientOptions) {
         try {
-          assert.strictEqual(options, clientOptions);
+          assert.deepStrictEqual(options, clientOptions);
           done();
```

Comment:
Changes in this file now only check to see values are equal because we are passing a copy of the options down now.